doc/neomake.txt

```diff
@@ -31,13 +31,13 @@ fantastic vim plugins such as syntastic and dispatch.
 ==============================================================================
 2. Commands                                                   *neomake-commands*
 
-                                                         *:Neomake* *:NeomakeFile*
+                                                       *:Neomake* *:NeomakeFile*
```

Comment:
But I can see that it's bad when being viewed raw.
Maybe there should be the same level of conceal chars on each line then, i.e. split `:Neomake! :NeomakeProject` into two lines?